database-connector.js

```diff
@@ -0,0 +1,37 @@
+const config = require("../config");
+const MongoClient = require('mongodb').MongoClient;
+
+const ConnectionString = `mongodb://${config.mongo_db_user}:${config.mongo_db_password}@${config.mongo_db_host}:${config.mongo_db_port}`;
```

Comment:
Can we initialize this ConnectionString in the constructor and pass either the entire config object or just the mongo db parameters into the constructor as parameters